.gitignore

```diff
@@ -1 +1,5 @@
 bin
+
+# IDEs
+.idea
+*.iml
```

Comment:
nit: add newline

Reply:
Added newline to the end of the file.

Reply:
I have extra question: normally, who will click "Resolve conversation"? you or I in this case for example?

Reply:
In a situation like this it doesn't matter because resolution is clear. In more complex cases it depends. Use common sense and allow reviewers time to validate changes where appropriate.